src/browser_action/crux.js

```diff
@@ -0,0 +1,61 @@
+import { Metric } from './metric.js';
+import { CRUX_API_KEY } from './secrets.js';
+
+
+export class CrUX {
+
+  static load(pageUrl) {
+    const urlHelper = new URL(pageUrl);
+    const url = urlHelper.href;
+    const origin = urlHelper.origin;
+
+    return CrUX.query({url}).catch(e =>{
+      console.warn('CrUX URL data unavailable', e);
+      // If URL data is unavailable, fall back to the origin.
+      return CrUX.query({origin});
```

Comment:
[Minor] A console warning noting CrUX URL data was unavailable might suggest a failure state, whilst we're actually 'recovering' here by falling back to the origin. Perhaps the warning should be explicit about the fallback?